modules/api/src/main/java/org/eclipse/xpanse/api/OrchestratorApi.java

```diff
@@ -261,10 +263,10 @@ public SystemStatus health() {
      * @return Status of the managed service.
      */
     @Tag(name = "Service", description = "APIs to manage the service instances")
-    @GetMapping("/service/{managedServiceName}")
+    @GetMapping("/service")
     @ResponseStatus(HttpStatus.OK)
-    public ServiceStatus state(@PathVariable("managedServiceName") String managedServiceName) {
-        return this.orchestratorService.getManagedServiceState(managedServiceName);
+    public ServiceStatus state(@RequestParam("id") String id) {
```

Comment:
Is it better to use service/{id} with @PathVariable("id") to keep a unified style？